.github/actions/promote-to-stable/action.yml

```diff
@@ -56,8 +56,14 @@ runs:
 
           echo "[DEBUG] - Promoting $ARCH testing artifacts to stable."
           for ARTIFACT in ${SRC_PATHS[@]}; do
-            echo "[DEBUG] - Promoting $ARTIFACT to stable."
-            jf rt cp $ARTIFACT $TARGET_PATH --flat=true
+            echo "[DEBUG] - Downloading $ARTIFACT from TESTING."
+            jf rt download $ARTIFACT --flat
```

Comment:
since you are already in a loop : `"noarch" "x86_64"`
you will reupload noarch packages in the second iteration ?

Reply:
indeed, removed the local ARCH from within the loop
added cleanup of localdir to make sure it is empty before next loop iteration